### DISCUSSION TITLE
any chance for mask2former?

### DISCUSSION THREAD

**Discussion by caxel-ap** [qubvel/segmentation_models.pytorch#1043] [General]
thank you so much for the v0.4.0 release which brings us pretrained segformer and so many timm transformer encoders. what a great accomplishment. any chance in the future we might see the same for mask2former?

**Comment by qubvel** [comment]
Hey @caxel-ap! I appreciate you like the latest release 🤗 

At the moment, I don't have plans to add mask2former, but I might consider it in the future or in case anyone wants to contribute it 😄 
Btw, there are `MaskFormer` and `Mask2Former` in `transformers` [[docs](https://huggingface.co/docs/transformers/model_doc/mask2former)]. In case you already familiar with `transformers` implementation can you tell what is missing there?

**Comment by caxel-ap** [comment]
Thanks for the reply! Understood that there are no plans. I am indeed familiar with and have used the huggingface transformers implementation, I just like the smp library much better and the ability to seamlessly switch out different encoders

**Comment by qubvel** [comment]
> I just like the smp library much better and the ability to seamlessly switch out different encoders

Great to hear that 😄 Can you open an issue? I will add a label and maybe we will find a superhero to add Mask2Former

**Comment by caxel-ap** [comment]
good thinking, here it is https://github.com/qubvel-org/segmentation_models.pytorch/issues/1044
